packages/app/src/app/overmind/effects/live/clients.ts

```diff
@@ -39,7 +39,7 @@ class CodeSandboxOTClient extends Client {
   }
 
   sendOperation(revision, operation) {
-    this.onSendOperation(revision, operation);
+    this.onSendOperation(revision, operationToElixir(operation.toJSON()));
```

Comment:
I wonder how `master` currently works without this 🤔. But nice to see it fixed now!

Reply:
Was it not opposite? I had merge conflict. I chose the wrong thing, it is handled up above, 2 sec

Reply:
This is fixed now